### PR TITLE
Parallel & Concurrent Destination Testing

### PR DESCRIPTION
**Pull request by chriszielinski**
The commits should be self-explanatory. Let me know if there are any issues!

### CODE REVIEW

File: Sources/XcbeautifyLib/String+CapturedGroups.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ extension String {
             return results
         }
 
-        let matches = regex.matches(in: self, range: NSRange(location:0, length: self.count))
+        let matches = regex.matches(in: self, range: NSRange(location:0, length: self.utf16.count))
```
Comment (thii) [inline Sources/XcbeautifyLib/String+CapturedGroups.swift:14]: Why do you need UTF16View's count?

File: README.md [hunk 2]
```diff
@@ -84,6 +84,12 @@ If you want `xcbeautify` to exit with the same status code as `xcodebuild`
 set -o pipefail && xcodebuild [flags] | xcbeautify
 ```
 
+For parallel and concurrent destination testing, it helps to use unbuffered I/O for _stdout_ and to redirect _stderr_ to _stdout_.
+
+```bash
+NSUnbufferedIO=YES xcodebuild [flags] 2>&1 | xcbeautify
```
Comment (thii) [inline README.md:90]: TIL.

File: Sources/XcbeautifyLib/String+CapturedGroups.swift (hunk 1 above)
Comment (chriszielinski) [reply Sources/XcbeautifyLib/String+CapturedGroups.swift:14]: To be Unicode-safe. `NSRegularExpression` ingests UTF-16 code units—the "NS" class prefix hints towards this.

**Review by thii — APPROVED** [review]

**Comment by thii** [comment]
Sorry for the long wait. This has been released in 0.6.0.

**Comment by thii** [comment]
@chriszielinski I've also sent you an invite to be a collaborator. Please join if you're interested 🙂

**Pull request merged**